lib/legacy/legacy-element-mixin.html

```diff
@@ -433,8 +459,8 @@
        * that contains an insertion point (`<slot>`) inside its subtree.
        */
       distributeContent() {
-        if (window.ShadyDOM && this.shadowRoot) {
-          this.shadowRoot.forceRender();
+        if (window.ShadyDOM) {
+          window.ShadyDOM.flush();
```

Comment:
This must have been missed in a merge conflict. I'll pull the master branch version.